### ISSUE TITLE
Higher quality WAV and OGG support, and chrome textures and mirrors for brushes

### ISSUE THREAD

**Issue by UnizoneDev** [ValveSoftware/halflife#3115]
What I (and a few others) want is for the GoldSource engine to support 44.1khz stereo WAV files, and perhaps the first step in supporting OGG files for music. Also, I would like [chrome texture](https://the303.org/tutorials/gold_mdl_chrome.htm) support for brushes, so as to make metallic textures shine, and maybe some reflective surface support as well, so that mirrors could be made.

This is all for now...

**Comment by Bandock** [comment]
If support for OGG were added, I think it would be great to support the Opus audio codec (especially, as of late) due to great audio quality at almost any bitrate.

**Comment by UnizoneDev** [comment]
> If support for OGG were added, I think it would be great to support the Opus audio codec (especially, as of late) due to great audio quality at almost any bitrate.

Good idea Bandock, but there's a problem for me that I would like someone from Valve to answer.

What if I wanted to make a standalone game on the Goldsource engine, would I have to ask Valve to license the engine in some way?

**Comment by Bandock** [comment]
> > If support for OGG were added, I think it would be great to support the Opus audio codec (especially, as of late) due to great audio quality at almost any bitrate.
> 
> Good idea Bandock, but there's a problem for me that I would like someone from Valve to answer.
> 
> What if I wanted to make a standalone game on the Goldsource engine, would I have to ask Valve to license the engine in some way?

That would be correct.

**Comment by BlackShadow** [comment]
> > If support for OGG were added, I think it would be great to support the Opus audio codec (especially, as of late) due to great audio quality at almost any bitrate.
> 
> Good idea Bandock, but there's a problem for me that I would like someone from Valve to answer.
> 
> What if I wanted to make a standalone game on the Goldsource engine, would I have to ask Valve to license the engine in some way?

I'm not from Valve but i do know the answer. If you going to make your mod commercial then yes, you do have to get license from Valve. If you're going to release it on Steam and it's non-commercial make sure there's no copyrighted material in your mod as well.

**Comment by sabianroberts** [comment]
> Also, I would like chrome texture support for brushes, so as to make metallic textures shine, and maybe some reflective surface support as well, so that mirrors could be made.

You forgot to mention bumpmaps ;)

I know modders have got bumpmaps to work, but it requires replacing some engine-side DLLs which Steam (or VAC) then replaces again with the original.

**Comment by UnizoneDev** [comment]
I was awaiting for the release of the GoldSource engine code, but if it's going to take some time, I understand.
